android/src/main/java/co/touchlab/droidcon/android/MainActivity.kt

```diff
@@ -54,6 +58,11 @@ class MainActivity: ComponentActivity(), KoinComponent {
 
         analyticsService.logEvent(AnalyticsService.EVENT_STARTED)
 
+        if (applicationViewModel.lifecycle.isAttached) {
+            applicationViewModel.lifecycle.removeFromParent()
+        }
```

Comment:
@TadeasKriz  After coming back to minimised app by clicking on a notification it crashed because the view model's lifecycle was already attached to a parent, is this a correct solution?